test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -932,17 +912,9 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox app %s', () => {
     const stackFrames = await Promise.all(
       stackFrameElements.map((f) => f.innerText())
     )
-    expect(stackFrames).toEqual(
-      // TODO: Show useful stack
-      [
-        // Internal frames of React.
-        // Feel free to adjust until we show useful stacks.
-        '',
-        '',
-        '',
-        '',
-      ]
-    )
+
+    // Filtered out the nodejs internal calls by default
```

Comment:
This isn't related to nodejs internals specifically. That's only incidental due to our current implementation. What's relevant here is that there is only 1 frame in product code and that's displayed in the codeframe. So this test needs to start asserting on the codeframe. Otherwise it looks like there is no stack at all which would be a bug.